### PR TITLE
Added templates to 'pull request' and 'issue'.

### PR DESCRIPTION
**Pull request by PauloGoncalvesBH**
Added templates to help the user to open a issue or create pull request.

 I think that's important to the project be supported by the community.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+[What you expected to happen]
+
+### Actual behavior:
+
+[What actually happened]
+
+### Versions
+
+[You can get this information from executing `npm list --depth=0`, copy and paste here]
```
Comment (wlsf82) [inline .github/ISSUE_TEMPLATE.md]: What about asking for the node (`node -v`) and npm (`npm -v`) versions too?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+ ## Pull Request Checklist
+
+* [ ] Have you followed the steps in our [Contributing](../../blob/master/docs/CONTRIBUTING.md#steps-to-contribute) document?
+* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
+* [ ] Have you successfully ran tests with your changes locally?
+* [ ] Have you written new tests for your changes, if necessary?
+* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
+* [ ] Have you made improvements to the documentation, if necessary? ([docs/](../../blob/master/docs/) folder and [README.md](../../blob/master/README.md))
+* [ ] Have you link the correct [GitHub issue](../../issues), if contains?
```
Comment (wlsf82) [inline .github/PULL_REQUEST_TEMPLATE.md]: Shouldn't it be 'linked', instead of 'link'
And, 'if there is any', instead of 'if contains'

Comment (PauloGoncalvesBH) [reply .github/PULL_REQUEST_TEMPLATE.md]: done.

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (PauloGoncalvesBH) [reply .github/ISSUE_TEMPLATE.md]: Done.

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+[What you expected to happen]
+
+### Actual behavior:
+
+[What actually happened]
+
+### Version of your dependencies, npm and node
+
+[You can get this information from executing the following 3 commands: `npm list --depth=0`, `npm -v` and `node -v`]
```
Comment (wlsf82) [inline .github/ISSUE_TEMPLATE.md]: Let's go with '[You can get this information by executing the following command: `npm list --depth=0 && npm -v && node -v`. Just copy and paste it, then hit RETURN in your CLI]'.

Comment (PauloGoncalvesBH) [reply .github/ISSUE_TEMPLATE.md]: Done.

File: .github/ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+[What you expected to happen]
+
+### Actual behavior:
+
+[What actually happened]
+
+### Version of your dependencies, npm and node
+
+[You can get this information by executing the following command: `npm list --depth=0 && npm -v && node -v`. Just copy and paste it, then hit RETURN in your CLI]
```
Comment (wlsf82) [inline .github/ISSUE_TEMPLATE.md]: I'm not sure if it does work for Windows OS. Can you confirm, please?

Comment (PauloGoncalvesBH) [reply .github/ISSUE_TEMPLATE.md]: That work, I tried.

File: .github/ISSUE_TEMPLATE.md (hunk 3 above)
Comment (PauloGoncalvesBH) [reply .github/ISSUE_TEMPLATE.md]: The 'hit RETURN' don't make sense for me. I think that's better:
[You can get this information by executing the following command: `npm list --depth=0 && npm -v && node -v`. Just copy and paste it]

**Pull request merged**